src/main/resources/io/federecio/dropwizard/swagger/index.ftl

```diff
@@ -64,7 +64,9 @@
             });
             */
           }
-
+          if(${isUiReadOnly?c}){
```

Comment:
When the number is printed not for human audience (e.g., for a database record ID used as the part of an URL, or as invisible field value in a HTML form, or for printing CSS/JavaScript numerical literals) this built-in must be used to print the number (i.e., use ${x?c} instead of ${x}), or else the output will be possibly broken depending on the current number formatting settings and locale (like the decimal point is not dot, but comma in many countries) and the value of the number (like big numbers are possibly ``damaged'' by grouping separators).